src/regex.h

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <sstream>
+#include <regex>
+#include <stdint.h>
+// #include <boost/regex.hpp>
+
+using namespace std;
+using namespace boost;
+
+bool isRegex(const string& txData) {
+    // Ef strengurinn byrjar á "ACCEPT offer "(í hex) 
+    // og ef strengurinn er 90 á lengd
+    // þá skoðum við txid sem kemur þar á eftir
+
+    LogPrintStr(" 1 i regex: "+txData);
+    if (txData.rfind("414343455054206f6666657220", 0) == 0 && txData.length() == 90) {
+        // náum í txid-ið eða þá 64 stafi sem eru á eftir "ACCEPT offer "
+        std::string txid = txData.substr(26, 64);
+        // Köllum á gettransaction með því txid - ef það er ekki til þá skilum við false
+        uint256 hash = ParseHashV(txid, "parameter 1");
+        CTransaction tx;
+        uint256 hashBlock = 0;
+        if (!GetTransaction(hash, tx, hashBlock, true))
+        {
+            return false;
+        }
+    }
+
+    std::vector<std::string> lines;
+    string filename = "/Users/freyja/Documents/mittSmiley/smileyCoin/src/file.txt"; // could come from command line.
```

Comment:
This probably needs to be changed, adding an option to open files from the cli is probably a good idea but the file itself should not live in the source tree nor be hardcoded to an absolute path.

Maybe it could be stored somewhere in `$HOME/.smileycoin/` ?

An outsider will have a hard time figuring out what this file does, maybe it could have a more descriptive name?